src/pages/cart/cart.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/extensions */
+import Total from '../../core/components/total';
+import Page from '../../core/templates/page';
+
+class CartPage extends Page {
+  // eslint-disable-next-line no-useless-constructor
+  constructor(id: string, className: string) {
+    super(id, className);
+  }
+
+  createCartHead() {
+    const cartHead = this.createWrapper('cart__head');
```

Comment:
better to use static method than protected, coz if u using this.createWrapper another developer trying to find it in this class, but if u use static it looks more obviously Page.createWrapper()